src/controls/TableMenuControls.tsx

```diff
@@ -137,6 +138,10 @@ export default function TableMenuControls({
 
       <MenuDivider />
 
+      <MenuSelectTableBorderStyle />
```

Comment:
Ideally the TableMenuControls can be used for a user who's included `Table` or `TableImproved`, with or without the new `TableCellImproved` and `TableHeaderImproved` extensions. We should allow these border style controls to be conditionally included, or dynamically include it automatically only if the appropriate extensions are present for the `editor`.